docs/reference/releases-servicing.md

```diff
@@ -7,9 +7,9 @@ nav_order: 1
 
 # ONNX Runtime releases
 
-The current ONNX Runtime release is [1.13](https://github.com/microsoft/onnxruntime/releases/tag/v1.13.1).
+The current ONNX Runtime release is [1.14](https://github.com/microsoft/onnxruntime/releases/tag/v1.14).
```

Comment:
1.14.0, and link is https://github.com/microsoft/onnxruntime/releases/tag/v1.14.0